core/src/main/resources/AppConstants.properties

```diff
@@ -93,14 +93,27 @@ browse.errors.order = ASC
 transactionmanager.log.dir=${log.dir}/transactionlogs/${instance.name.lc}
 transactionmanager.uid=
 
+# Maximum timeout (in s) that can be allowed for transactions.
+transactionmanager.defaultTransactionTimeout=180
+
+# Amount of time (in ms) between runs of the TX maintenance thread.
+transactionmanager.narayana.reapTime=120000
```

Comment:
Je zet nu deze reap time op 2 minuten, alsof het de transactie timeout is. Voor zover ik begrijp is het dat niet, het is alleen maar het interval waarop het thread loopt dat transacties opruimt die opgeruimd moeten worden. Het is dan niet zinvol om die tijd heel hoog te zetten. Dan blijven die op te ruimen transacties langer hangen dan noodzakelijk.
Of heb ik het mis? Kan je aangeven waarom je deze op 2 minuten zet?

Reply:
120000ms is de default in Narayana.

Reply:
Is het nodig om de reap-time te zetten, en de andere Narayana tweaks? Wat brengen ze, naast het verhogen van de default-timeout?

Reply:
De andere tweaks doen niets, behalve het voor ontwikkelaars configureerbaar maken deze waardes aan te passen.